server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -229,6 +229,25 @@ export function create(
 		},
 	);
 
+    /**
+     * Delete a document
+     */
+    router.post(
+        "/:tenantId/document/:id/delete",
```

Comment:
If you use `router.delete`, the path could be just `"/:tenantId/document/:id`

Reply:
updated to use router.delete